discovery-provider/src/utils/helpers.py

```diff
@@ -176,3 +178,29 @@ def update_ipfs_peers_from_user_endpoint(update_task, cnode_url_list):
 
 latest_block_redis_key = 'latest_block_from_chain'
 latest_block_hash_redis_key = 'latest_blockhash_from_chain'
+
+# Constructs a track's route_id from an unsanitized title and handle.
+# Resulting route_ids are of the shape `<handle>/<sanitized_title>`.
+def create_track_route_id(title, handle):
+    # Strip out invalid character
+    sanitized_title = re.sub(r'!|%|#|\$|&|\'|\(|\)|&|\*|\+|,|\/|:|;|=|\?|@|\[|\]', '', title)
```

Comment:
jw, do non-unicode characters play nicely with this? like "ℂ𝕒𝕟'𝕥 𝕔𝕙𝕒𝕟𝕘𝕖 𝕞𝕪 𝕙𝕒𝕟𝕕𝕝𝕖. 𝕀 𝕚𝕕𝕖𝕟𝕥𝕚𝕗𝕪 𝕒𝕤 𝕍𝕒𝕕𝕖𝕣 𝕟𝕠𝕨"

Reply:
Tested in a REPL, seems fine. Good callout.